Monika After Story/game/script-affection.rpy

```diff
@@ -2372,7 +2372,7 @@ init python:
             filepath = "/My one and only love.txt"
             message = """\
 My dearest lover, friend, companion and owner of my heart...
-Everyday you make my dreams come true, a screen means nothing when you spend your time with me.
+Every day you make my dreams come true, a screen means nothing when you spend your time with me.
```

Comment:
```suggestion
Every day, you make my dreams come true, a screen means nothing when you spend your time with me.
```

Reply:
One thing to make this a bit more correct.

Reply:
Approved